keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ApplicationSummaryGenerationTests.kt

```diff
@@ -95,15 +94,15 @@ abstract class ApplicationSummaryGenerationTests<T : ArtifactRepository> : JUnit
       }
 
       test("skipped versions don't get a pending status in the next env") {
-        val summaries = subject.getEnvironmentSummaries(manifest).sortedBy { it.name }
+        val envSummaries = subject.getEnvironmentSummaries(manifest).sortedBy { it.name }
         expect {
-          that(summaries.size).isEqualTo(2)
-          that(summaries[0].artifacts.first().versions.current).isEqualTo(version2)
-          that(summaries[0].artifacts.first().versions.pending).isEmpty()
-          that(summaries[0].artifacts.first().versions.skipped).containsExactlyInAnyOrder(version1)
-          that(summaries[1].artifacts.first().versions.current).isEqualTo(version2)
-          that(summaries[1].artifacts.first().versions.pending).isEmpty()
-          that(summaries[1].artifacts.first().versions.skipped).containsExactlyInAnyOrder(version1)
+          that(envSummaries.size).isEqualTo(2)
+          that(envSummaries[0].artifacts.first().versions.current).isEqualTo(version2)
+          that(envSummaries[0].artifacts.first().versions.pending).isEmpty()
+          that(envSummaries[0].artifacts.first().versions.skipped).containsExactlyInAnyOrder(version1)
```

Comment:
If the list has a single element you could just use `containsExactly()`, right?

Reply:
Yes